core/frontend/src/menus.ts

```diff
@@ -46,33 +33,46 @@ const menus = [
               + ' currently only supported for Navigator boards.',
       },
       {
-        title: 'Endpoints',
-        icon: 'mdi-arrow-decision',
-        route: '/vehicle/endpoints',
+        title: 'Network Test',
```

Comment:
Are you aiming for an alphabetical ordering here? If so, `MAVLink Inspector` should be before `Network Test`.

Reply:
Thanks!